hermetic_build/library_generation/owlbot/templates/java_library/.kokoro/build.sh

```diff
@@ -1,12 +1,11 @@
 #!/bin/bash
-
-# Copyright 2018 Google LLC
+# Copyright 2025 Google LLC
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#     https://www.apache.org/licenses/LICENSE-2.0
+#      https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Is this change intended?

Reply:
Oh, my bad. It's not. I just fixed it, thanks!